distributed/tests/test_zstd.py

```diff
@@ -0,0 +1,52 @@
+def test_zstd():
+    import zstandard, zstandard as zstd, six, cffi, dask, distributed.protocol.compression as compr
```

Comment:
we probably want to skip this or the entire module if zstd is not installed

Reply:
@crepererum  

commit 00703010e569a842415c25389fa514d2b2e95c06 (HEAD -> patch-5, origin/patch-5)
Author: heyijun <heyijun@qianxin.com>
Date:   Fri Aug 23 21:09:00 2019 +0800

     ONLY call test_zstd() if "zstandard" installed